subscription-sender/internal/queue/notification.go

```diff
@@ -2,15 +2,14 @@ package queue
 
 import (
 	"context"
-	"encoding/json"
 
-	"github.com/RinaDish/subscription-sender/internal/services"
 	"github.com/nats-io/nats.go"
 )
 
 func (queue *Queue) Subscribe(ctx context.Context) error {
 	_, err := queue.QueueConn.Subscribe(queue.subscriptionTopicName, func(msg *nats.Msg) {
-        _ = queue.handleMessage(msg, ctx)
+		queue.logger.Infof("Received a message: %s\n", string(msg.Data))
+        queue.messagesService.GetMessage(msg, ctx)
```

Comment:
IMO GetMessage is confusing naming for this method as MessageService does not returns any messages, but processes them.

Reply:
it's handleMessage already